server/master/src/main/java/org/apache/accumulo/master/FateServiceHandler.java

```diff
@@ -664,4 +670,41 @@ private String validateNamespaceArgument(ByteBuffer namespaceArg, TableOperation
           TableOperationExceptionType.INVALID_NAME, why);
     }
   }
+
+  /**
+   * Create a file on the file system to hold the splits to be created at table creation.
+   */
+  private String createSplitFile(final long opid, final List<ByteBuffer> arguments,
+      final int splitCount, final int splitOffset) {
+    String opidStr = String.format("%016x", opid);
+    String splitPath = getSplitPath("/tmp/splits-" + opidStr);
+    try (FSDataOutputStream stream = master.getOutputStream(splitPath)) {
+      writeSplitsToFileSystem(stream, arguments, splitCount, splitOffset);
+    } catch (IOException e) {
+      splitPath = null;
+      e.printStackTrace();
```

Comment:
Seems like this should log a message (with filename, error, and opid) and maybe rethrow the exception which should cause the create table op to fail.

Reply:
Added a log message and rethrow. Table Op will fail with ThriftTableOperationException if write fails.